### ISSUE TITLE
Update Getting Started page

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1413]
### Overview
We need to add information about the Communities of Practice to the Getting Started page so that it correctly reflects HfLA's onboarding

### Action Items
- [x] Review the updated [Getting Started page](https://www.hackforla.org/getting-started) design on [Figma (in purple rectangle)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13%3A0) - added third 'Communities of Practice' section to Step 1 page card
- [x] Add the Communities of Practice section to bottom of Step 1 on the Getting Started Page 
- [x] Add the new icon's [credit info](https://github.com/hackforla/website/wiki/Template-of-a-credit.md-file-(for-Credits-page)) to the [Credits page](https://www.hackforla.org/credits/)

### Resources/Instructions
- Updated Getting Started page design on [Figma (in purple rectangle)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13%3A0)
- [Getting Started page](https://www.hackforla.org/getting-started)
- [Credits page](https://www.hackforla.org/credits/)
- [Credit template](https://github.com/hackforla/website/wiki/Template-of-a-credit.md-file-(for-Credits-page))

New Getting Started page section:

<img width="450" alt="Screen Shot 2021-04-18 at 3 11 43 AM" src="https://user-images.githubusercontent.com/67400366/115141948-f323a180-9ff3-11eb-9d56-e6251573341e.png">

**Comment by RobertaFricker** [comment]
@daniellex0   Hi Danielle, Akib asked me to start working on this issue.  I found the image in Figma, however, I'm not sure how to copy it correctly.  Here's a screenshot of what it looks like in Figma.  How do I copy all of the sections so that the hyperlinks work, etc...:
![image](https://user-images.githubusercontent.com/81199122/115942155-b0cbeb80-a45d-11eb-988c-a712e6c53626.png)

Also, here's a rough draft of where I think you want it to go on the Getting Started page.  Please disregard the size, position, etc...it's just the basic idea:

![image](https://user-images.githubusercontent.com/81199122/115942201-e40e7a80-a45d-11eb-882f-d3485016cac0.png)

Please advise on these fun points, thanks!

**Comment by daniellex0** [comment]
Hi @RobertaFricker I'm not sure I fully understand- I think if you follow the lead of the other similar sections in the [Getting Started page](https://www.hackforla.org/getting-started), it should hopefully be pretty straightforward. But if you're trying to figure out how to code it, it might be best to consult with the devs at tomorrow's meeting..

**Comment by RobertaFricker** [comment]
Hi @daniellexo I'm unclear how to copy the image from the Figma page.  You
cannot just copy it as a whole image, it's in layers.  If I copy it as an
svg, it pastes in code format.
The image on the 1413 issue page doesn't retain the hyperlinks, it's just a
basic .png image.  If I copy and paste it and then hover over the
hyperlinks, it doesn't show a hand, it shows the 4-arrow symbol.

**Comment by daniellex0** [comment]
Oh hmm that's odd... that might be good to ask the devs today as well... Have you tried "Export" the image via the right column?

FYI also here is a (draft) guide for Figma for developers, in case it's useful: https://drive.google.com/file/d/1Is5Ah0tUH07kWYG_J6HTo0RF8Z9Foi2l/view?usp=sharing

**Comment by daniellex0** [comment]
Have you tried "Export" the image via the right column?

**Comment by qiqicodes** [comment]
@jbubar pair programming with @RobertaFricker tomorrow to work this issue.

**Comment by RobertaFricker** [comment]
Status update:
Blockers:  Need assistance on how to add image that consists of various layers into the existing .html
Questions:  Should I copy each layer separately from Figma?  How do I get the same properties from the Figma images?

**Comment by RobertaFricker** [comment]
Status and Blocker:
I have successfully copied 3 svg images for the Communities of Practice section.  I've added two of them to test.  I'm having issues getting space in between them per the localhost screenshot below:
![image](https://user-images.githubusercontent.com/81199122/116636622-8c1cbb80-a916-11eb-9d40-8e84c7a5a84a.png)

I wasn't able to add the svg using the `img` tag if it's in the _includes/svg folder per screenshot below:
![image](https://user-images.githubusercontent.com/81199122/116636770-f7668d80-a916-11eb-88cf-48d8d6cf6290.png)
It would have to be in the assets/image....folder to work with an `img` tag.

I had to add it using the `{% include   %} ` syntax which is the same that's used in header.html among other pages:
![image](https://user-images.githubusercontent.com/81199122/116636903-4f04f900-a917-11eb-8533-aacdabcb6479.png)

`<div class="project-position-2"  >
                  
                     {% include svg/icon-comm-of-prac-discu.svg  %}

                <div class="project-position-2"  >    
                   
                     {% include svg/join-your-comm-of-prac.svg %}`

Whoever can assist with moving me forward would be appreciated, thanks!
